spec/requests/api/journalist_can_create_an_article.spec.rb

```diff
@@ -0,0 +1,43 @@
+RSpec.describe 'POST/api/articles', type: :request do
+  let!(:articles) { create(:article) }
```

Comment:
why are you creating an article?

Reply:
Dont we need it to create an Article in the test?

Reply:
it is removed now btw

Reply:
@Ehugo2000 only if you want to return an article, in this case you want to create it.